redisinsight/ui/src/components/base/forms/combo-box/AutoTag.tsx

```diff
@@ -220,3 +221,11 @@ const StyledWrapper = styled(Row)`
   background-color: ${({ theme }) =>
     theme.semantic.color.background.neutral100};
 `
+
+const FlexibleInputWrapper = styled.div`
+  flex: 1;
+  min-width: 50px;
+  > * {
+    width: 100%;
+  }
+`
```

Comment:
Instead of creating a wrapper, you can add the styles to the input:
```suggestion
const StyledInput = styled(Input)`
  flex: 1;
  min-width: 30px;
`
```

Also, please define it at the top of the follow, not the bottom. Use it like this (almost same as now):
```
           <StyledInput
            variant="underline"
            placeholder={getPlaceholder()}
            onChange={handleInputChange}
            onKeyDown={handleEnter}
            value={tag}
            data-test-subj="autoTagInput"
          />
```